src/xtopcom/xcontract_common/src/xcontract_state.cpp

```diff
@@ -13,14 +13,21 @@
 
 NS_BEG2(top, contract_common)
 
-xtop_contract_state::xtop_contract_state(common::xaccount_address_t action_account_addr,
-                                         observer_ptr<state_accessor::xstate_accessor_t> sa,
+xtop_contract_state::xtop_contract_state(common::xaccount_address_t const & action_account_addr,
```

Comment:
`action_account_addr` will be `move`d in initialize list `m_action_account_address{std::move(action_account_addr}`, thus action_account_addr shouldn't be marked as `const &`